app/view/panel/application/StartViewController.js

```diff
@@ -30,14 +41,18 @@ Ext.define('MoMo.admin.view.panel.application.StartViewController', {
                 })
             ],
             view: new ol.View({
-                center: ol.proj.fromLonLat([7.1067575, 50.7374741]),
-                zoom: 12
+                center: ol.proj.fromLonLat([x, y]),
+                projection: ol.proj.get(projCode),
+                zoom: zoom
             })
         });
 
         view.down('gx_map').setMap(olMap);
     },
 
+    /**
+     * 
+     */
     addMapCrossHair: function() {
```

Comment:
Do we still need this function? You've delete the call [above](https://github.com/terrestris/momo3-admin/pull/10/commits/1850e842f3da9e7015bbf2929e74e5753268ec69#diff-188c4c06c7ac4067f2d0d2e2f9b07670L9)